sources/iTermAdvancedSettingsModel.m

```diff
@@ -151,4 +151,11 @@ + (NSString *)name { \
 
 DEFINE_BOOL(noSyncTipsDisabled, NO, @"Tip of the Day: Disable the Tip of the Day?");
 
+#pragma mark - Badge
+DEFINE_STRING(badgeFont, @"Helvetica", @"Badge: Font to use for the badge.");
+DEFINE_BOOL(badgeFontIsBold, YES, @"Badge: Make the badge font bold.");
+DEFINE_FLOAT(badgeMaxWidthPercent, 0.5, @"Badge: Maximum width of the badge\nAs a percentage of the width of the terminal, between 1% and 100%.");
```

Comment:
I had limited it to 0.01. I guess I was thinking to avoid a user "losing" the badge altogether, but it's not really necessary.

I think I've literally lost the ability to see 0.2 and 20% as different over the years.